### DISCUSSION TITLE
Need Assistance Syncing the Flowise API with Bubble.io for an Excel Formula Bot

### DISCUSSION THREAD

**Discussion by netanelyasi** [FlowiseAI/Flowise#254] [Q&A]
I'm developing a bot that generates Excel formulae depending on user input. However, I'm having trouble synchronizing the Flowise API with the Bubble.io API connection, which results in a permanent "#N/A" answer.

Your enlightening YouTube tutorials have been a huge help in my project, and I feel your knowledge can assist me overcome this obstacle. For your convenience, photos of the process and API settings are included.

Attached images: Flow connector + API bubble.

Any help you can give would be highly appreciated.

Thank you in advance for your consideration.

![IMG-20230604-WA0061.jpg](https://github.com/FlowiseAI/Flowise/assets/126358650/88f813da-94fc-4be6-824b-6c2942eef49d)

![IMG-20230604-WA0062.jpg](https://github.com/FlowiseAI/Flowise/assets/126358650/13915ce4-74eb-4bed-9e40-6700ff53853f)

**Comment by HenryHengZJ** [comment]
hey! does it work if you try removing the overrideConfig?

**Reply by netanelyasi** [reply]
I tried to play around with it a bit: let's say if I use a translator template that works. But if I use few shot because it asks for a template like this

[
  {"input": "example", "output": "example"} 
]

Bubble.io api connector Does not recognize the patterns.

i.e. in flowise i need certain JSON pattern with spaces bubble.io can't identify and I get N/A
